x-pack/plugins/dataset_quality/server/routes/register_routes.ts

```diff
@@ -71,8 +73,12 @@ export function registerRoutes({ repository, core, logger, plugins }: RegisterRo
               body: { message: error.output.payload.message },
             });
           }
-
           logger.error(error);
+
+          if (isKibanaResponse(error)) {
```

Comment:
Is this code reachable? I think we will always return when we encounter an error in the above `if`

Reply:
After using Boom, this has been removed now.